arroyo-rpc/proto/api.proto

```diff
@@ -636,6 +642,7 @@ message CreateSourceReq {
     ImpulseSourceConfig impulse = 4;
     FileSourceConfig file = 5;
     NexmarkSourceConfig nexmark = 6;
+    KafkaSourceConfig raw_kafka = 7;
```

Comment:
Why is it necessary to have raw_kafka be separate from kafka here? Would you expect to also have a proto_kafka and avro_kafka, and proto_pulsar, etc.?